test/e2e_external/dataplane/connectivity.go

```diff
@@ -0,0 +1,320 @@
+/*
+© 2021 Red Hat, Inc.
```

Comment:
I guess this needs update :D

Reply:
I guess it's okay to just not have this line? The linter is passing, I guess because we don't check for this part in `.legal.json`.